.drone.star

```diff
@@ -796,6 +796,7 @@ def acceptance(ctx):
 		'visualTesting': False,
 		'openIdConnect': False,
 		'oc10IntegrationAppIncluded': False,
+		'skip': False,
```

Comment:
Do we want to have this "CI feature" in other repos - oC10 core, ocis, reva, oC10 apps?
It will be easy for someone to add it to `.drone.star` and then we document it.